examples/chip-tool/entitlements/chip-tool.entitlements

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wonder whether it would make sense to have only one copy of the identical entitlements file...

Reply:
Followup